.babelrc

```diff
@@ -0,0 +1,28 @@
+{
+  "plugins": [
+    [
+      "@babel/plugin-proposal-decorators",
+      {
+        "legacy": true
+      }
+    ],
+    [
+      "@babel/plugin-proposal-class-properties",
+      {
+        "loose": true
+      }
+    ],
+  ],
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            "last 2 Chrome versions"
```

Comment:
no love for IE ? 😆

Reply:
not at the beginning. targeting chrome, then firefox. With the new Edge using chromium as base, shouldn't theoretically it work? not sure how their extensions works.

Reply:
JK